3rdParty/qxt/qxtglobalshortcut.cpp

```diff
@@ -35,9 +35,6 @@
 
 #ifndef Q_OS_MAC
 int QxtGlobalShortcutPrivate::ref = 0;
-#   if QT_VERSION < QT_VERSION_CHECK(5,0,0)
-QAbstractEventDispatcher::EventFilter QxtGlobalShortcutPrivate::prevEventFilter = 0;
-#   endif

```

Comment:
I'm not sure if you made these changes or took them from the source, but I would suggest that if possible, we update qxt from https://github.com/hluk/CopyQ/tree/master/qxt (at least, this was the original source - 0a2b49526dcb1e95c805c4efeb97172d05049fc6) in a separate PR.